data/Templates/Ccda/ValueSet/_AddressUse.liquid

```diff
@@ -10,5 +10,5 @@
     {% when "BAD" -%}
         "old",
     {% else -%}
-        "{{ code }}",
+        "temp",
```

Comment:
Why temp? Who not leave the original value so that users can translate it in post processing?

Reply:
Per the offline discussion, an empty string will be used for the unmapped cases in this pr. And creating extensions to handle it later.